### ISSUE TITLE
StackLayout Tutorial Step 1 Feedback

### ISSUE THREAD

**Issue by 110zhengyufeng** [MicrosoftDocs/xamarin-docs#2620]
### Description

Severity	Code	Description	Project	File	Line	Suppression State
Error		Failed to create JavaTypeInfo for class: Android.Support.V4.View.Accessibility.AccessibilityManagerCompat/IAccessibilityStateChangeListenerImplementor due to MAX_PATH: System.IO.DirectoryNotFoundException: Could not find a part of the path 'D:\YFIShareFolder\Public\SourceCode\Xamarin\Samples\StackLayoutTutorial\StackLayoutTutorial\StackLayoutTutorial.Android\obj\Debug\90\android\src\mono\android\support\v4\view\accessibility\AccessibilityManagerCompat_AccessibilityStateChangeListenerImplementor.java'.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.File.InternalDelete(String path, Boolean checkHost)
   at System.IO.File.Delete(String path)
   at Xamarin.Android.Tools.Files.CopyIfStreamChanged(Stream stream, String destination)
   at Xamarin.Android.Tasks.Generator.CreateJavaSources(TaskLoggingHelper log, IEnumerable`1 javaTypes, String outputPath, String applicationJavaClass, String androidSdkPlatform, Boolean useSharedRuntime, Boolean generateOnCreateOverrides, Boolean hasExportReference)	StackLayoutTutorial.Android			

### Information

- IDE and IDE version: 
- Platform with the issue: 
  - iOS:  
  - Android:

**Comment by 110zhengyufeng** [comment]
Has fixed Refer to below 
https://developercommunity.visualstudio.com/content/problem/521034/failed-to-create-javatypeinfo.html
